### PR TITLE
Closes #2645: Add events to Quickstart for Marketing Cloud module

### LINKED ISSUE #2645: Add events to Marketing Cloud module
Follow up to https://github.com/az-digital/az_quickstart/issues/2327

## Motivation
I want to copy HTML for an event to put into Marketing Cloud.

## Proposed Resolution
Add events to Content for email new module in the same fashion News content is added/displayed
[Figma mockup for Quickstart](https://www.figma.com/file/KBr2TQ5YVGimwErjfMk4Xv/QS2-News-in-Marketing-Cloud?type=design&node-id=188-299&mode=design&t=McdIQRTpWJHZtxyo-0)
[Codepen for Marketing Cloud HTML](https://codepen.io/itsmemario/pen/GRXNPRL)

### PR DESCRIPTION
**Pull request by danahertzberg**
## Description
Need help with adding link to title

## Related issues

Closes #2645 

## How to test

1. Enable the Quickstart for Marketing Cloud module
2. Add events to the website
3. View new Content for email page
4. Click on Event layout for an event
5. View event rendered for email
6. Verify link goes to site or external link

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [x] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_marketing_cloud/templates/node--view--az-marketing-cloud--event-layout.html.twig [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                    </tr>
+                  </tbody>
+                </table>
+
+              </td>
+              <td valign="top" style="width: 85%; padding-left: 7.5px;">
+
+                <!-- QS: TITLE & LINK (recomended 80 character max) -->
+                <p style="line-height: 130%; margin-top:0;" class="qs-heading">
+                  <a class="qs-link" href="{{ az_canonical_url }}" style="color:#8B0015; font-weight: 700; text-decoration:underline;">
```
Comment (danahertzberg) [inline modules/custom/az_marketing_cloud/templates/node--view--az-marketing-cloud--event-layout.html.twig:106]: Look to see if az_canonical_url is applying to events and news

Comment (danahertzberg) [reply modules/custom/az_marketing_cloud/templates/node--view--az-marketing-cloud--event-layout.html.twig:106]: https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_marketing_cloud/az_marketing_cloud.module#L73

Comment (danahertzberg) [reply modules/custom/az_marketing_cloud/templates/node--view--az-marketing-cloud--event-layout.html.twig:106]: Look into if link goes to Trellis event (external) or QS event for Trellis events

Comment (danahertzberg) [reply modules/custom/az_marketing_cloud/templates/node--view--az-marketing-cloud--event-layout.html.twig:106]:
```
  $content_type = $node->getType();
  if (in_array($content_type, ['az_news', 'az_event'])) {
```

**Comment by trackleft** [comment]
I tested the external link functionality: https://7eb03a2b-3c7a-468f-872f-ac9ca87532b3.probo.build/export/marketing-cloud/event-layout/34 Seems to work.

**Review by ejsamboy — APPROVED** [review]
I followed the steps below to test this, and everything seems to work well.

Enable the Quickstart for Marketing Cloud module
Add events to the website
View new Content for email page
Click on Event layout for an event
View event rendered for email
Verify link goes to site or external link

**Comment by trackleft** [comment]
See https://github.com/az-digital/az_quickstart/issues/2671
See https://github.com/az-digital/az_quickstart/issues/2672

**Review by trackleft — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]

**Pull request merged**